linux/proc/init_state.go

```diff
@@ -193,32 +190,6 @@ func (s *createdCheckpointState) Resize(ws console.WinSize) error {
 func (s *createdCheckpointState) Start(ctx context.Context) error {
 	s.p.mu.Lock()
 	defer s.p.mu.Unlock()
-	p := s.p
```

Comment:
Is there a way to block criu from starting the final container process until we signal it?

Reply:
@crosbymichael Could you explain the idea of this barrier?

Reply:
It's to mimic the create/start split so there is time after the namespaces and process has been created before the user's process starts.